umbraco-deploy/deploy-settings.md

```diff
@@ -260,3 +263,27 @@ If you would prefer to use SQL Server LocalDb when it's available on your local
         }
     }
 ```
+
+## MediaFileChecksumCalculationMethod
+
+Deploy will do comparisons between the entities in different environments to determine if they match and decide whether to include them in the operation. By default, for media files, a check is made on the a portion of the intial bytes of the file.
```

Comment:
The line reads: By default, for media files, a check is made on **the** **a** portion of the intial bytes of the file.

I am not sure, whether it should be: **a portion** or **the portion?** 🤔

Reply:
"a portion".  I've fixed it.